### PR TITLE
Add Number.prototype.toFixed return value test

### PR DESCRIPTION
**Pull request by kadamwhite**
Adds a test to ensure that toFixed returns a string value

### CODE REVIEW

**Comment by rwaldron** [comment]
Note to other maintainers: @kadamwhite is employed by Bocoup, which is an Ecma member company.

File: test/built-ins/Number/prototype/toFixed/return-type.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+// Copyright (C) 2017 K. Adam White. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+es6id: 20.1.3.3
```
Comment (rwaldron) [inline test/built-ins/Number/prototype/toFixed/return-type.js]: This line can be removed. Once that's complete, this will be merged.

**Comment by kadamwhite** [comment]
Removed the line as requested, can rebase PR if needed

**Comment by leobalter** [comment]
LGTM

**Comment by rwaldron** [comment]
Thanks @leobalter

**Pull request merged**